jbpm-wb-integration/jbpm-wb-integration-client/src/main/java/org/jbpm/workbench/wi/client/editors/deployment/descriptor/items/NamedObjectItemView.java

```diff
@@ -46,7 +48,7 @@
 
     @Inject
     @DataField("resolvers")
-    private HTMLDivElement resolversContainer;
+    private KieEnumSelectElement<Resolver> resolversContainer;
```

Comment:
Please, give a proper name. `resolversSelect` might be a good name.

Reply:
Thanks for the review Pere, I have renamed the fields.